apps/fz_http/lib/fz_http/allow_rules.ex

```diff
@@ -32,6 +75,18 @@ defmodule FzHttp.AllowRules do
   end
 
   def list_allow_rules(%User{} = user) do
-    Repo.all(from r in AllowRule, where: r.user_id == ^user.uuid)
+    Repo.all(user_query(user.uuid))
+  end
+
+  def count do
+    Repo.aggregate(AllowRule, :count)
+  end
+
+  defp user_query(user_uuid) do
+    from(r in AllowRule, where: r.user_id == ^user_uuid)
+  end
+
+  def count(user_uuid) do
```

Comment:
It's better to name functions in a way it reflects it's action and arguments, eg:
```suggestion
  def count_by_user_id(user_id) do
```

This would allow us to extend the module eg. once we want to know how many allow rules are set for a `User.Group` (we have a task to add it somewhere in our backlog).